connector/connect/src/main/scala/org/apache/spark/sql/connect/dsl/package.scala

```diff
@@ -227,6 +227,21 @@ package object dsl {
       }
     }
 
+    implicit class DslStatFunctions(val logicalPlan: Relation) {
```

Comment:
does this need to be an implicit class?

Reply:
seems not needed, but widely used in this file and [sql/catalyst/dsl/package](https://github.com/apache/spark/blob/ff7ab34a6957965d74504ed1d36de21d1ad7319c/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/dsl/package.scala#L374-L386)